src/client/Health/NodeView.jsx

```diff
@@ -44,7 +44,7 @@ const NodeView = ({ status, onRestart, onShutdown }) => {
         label: (
           <div className={`oi-box tight text-${application.enabled ? 'success' : 'muted'}`}>
             <Link to={`/north/${application.applicationId}`}>
-              <div style={{ overflow: 'hidden', textOverflow: 'ellipsis' }}>{application.applicationId}</div>
+              <div id="overlay">{application.applicationId}</div>
```

Comment:
Why do you use an id ? Id should be use uniquely, but here you will have several id in your code with the same reference. What you need is a class, not an id